catalog/app/components/Lock/Lock.tsx

```diff
@@ -0,0 +1,25 @@
+import cx from 'classnames'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+import { fade } from '@material-ui/core/styles'
+
+const useStyles = M.makeStyles((t) => ({
+  root: {
+    background: fade(t.palette.background.paper, 0.5),
+    bottom: 0,
+    left: 0,
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    zIndex: 1,
+  },
+}))
+
+interface LockProps extends M.BoxProps {
```

Comment:
BoxProps arent used here

Reply:
Changed my mind, and forgot to remove it, thanks